File: codeforces/low_difficulty/difficulty_1000/1.theatre_square_[1A]/theatre_square.c
#include <stdio.h>
#include <math.h>
/* Author: José Rodolfo (jric2002) */
int main() {
  int n, m, a;
  long long int r;
  scanf("%i %li %i", &n, &m, &a);
  r = ceil(n / (double)a) * ceil(m / (double)a);
  printf("%lli\n", r);
  return 0;
}

File: codeforces/low_difficulty/difficulty_1000/1.theatre_square_[1A]/TheatreSquare.java
/* Author: José Rodolfo (jric2002) */
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.lang.Math;
public class TheatreSquare {
  public static void main(String[] args) throws IOException {
    Integer n, m, a;
    Double r;
    String[] values;
    BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
    values = (input.readLine()).split(" ");
    n = Integer.valueOf(values[0]);
    m = Integer.valueOf(values[1]);
    a = Integer.valueOf(values[2]);
    r = Math.ceil(n / a.doubleValue()) * Math.ceil(m / a.doubleValue());
    System.out.format("%.0f", r);
  }
}